### PR TITLE
Update Ray.origin reference

### PR DESCRIPTION
**Pull request by vilnytskyi**

### CODE REVIEW

**Review by Mugen87 — APPROVED** [review]

File: docs/api/en/math/Ray.html [hunk 1]
```diff
@@ -118,7 +118,7 @@ <h3>[method:Boolean equals]( [param:Ray ray] )</h3>
 		<p>
 		[page:Ray ray] - the [page:Ray] to compare to.<br /><br />
 
-		Returns true if this and the other [page:Ray ray] have equal [page:.offset offset]
+		Returns true if this and the other [page:Ray ray] have equal [page:.origin origin]
```
Comment (Mugen87) [inline docs/api/en/math/Ray.html:121]: Can you please also fix this error in the Chinese version of the page 😇 ?

https://threejs.org/docs/index.html#api/zh/math/Ray.equals

**Comment by Mugen87** [comment]
> Can you please also fix this error in the Chinese version of the page 😇 ?

Let me fix this 😊 .

**Comment by Mugen87** [comment]
1a8682136eda6cb474881c253f64f74f68a6bbfc

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**